server/client.go

```diff
@@ -1071,12 +1077,20 @@ func (c *client) processSub(argo []byte) (err error) {
 	}
 
 	// Check permissions if applicable.
-	if !c.canSubscribe(sub.subject) {
-		c.mu.Unlock()
-		c.sendErr(fmt.Sprintf("Permissions Violation for Subscription to %q", sub.subject))
-		c.Errorf("Subscription Violation - User %q, Subject %q, SID %s",
-			c.opts.Username, sub.subject, sub.sid)
-		return nil
+	if c.typ == ROUTER {
+		if !c.canExport(sub.subject) {
+			c.mu.Unlock()
+			c.Debugf("Ignoring subscription from route on %q due to export permissions", sub.subject)
```

Comment:
Its an error for client, not sure it should be for route but only debug seems like it might cause some support issues. WDYT?

Reply:
Agreed.. was more for me when coding, but not needed.